linkerd/protocol/http/src/main/scala/com/twitter/finagle/buoyant/linkerd/LinkerdHeaders.scala

```diff
@@ -473,7 +491,10 @@ object Headers {
   object Err {
     val Key = Prefix + "err"
 
-    def respond(msg: String, status: Status = Status.InternalServerError, maxHeaderSize: Int): Response = {
+    private[this] val log = Logger.get("FramingFilter.ServerFilter")
```

Comment:
I don't think this is in FramingFilter

Reply:
removed the log val, was needed only for debugging